drivers/docker/driver_test.go

```diff
@@ -1065,8 +1065,8 @@ func TestDockerDriver_CreateContainerConfig_Labels(t *testing.T) {
 	expectedLabels := map[string]string{
 		// user provided labels
 		"user_label": "user_value",
-		// default labels
-		"com.hashicorp.nomad.alloc_id": task.AllocID,
+		// default label
+		"com.hashicorp.nomad.alloc_id":        task.AllocID,
```

Comment:
I'm not sure why gofmt isn't picking this extra space up, but can we get rid of that?